drivers/scheduler/k8s/specs/postgres-pso/pure/pure-storage-class.yaml

```diff
@@ -0,0 +1,12 @@
+kind: StorageClass
+apiVersion: storage.k8s.io/v1
+metadata:
+  name: pure-sc
+parameters:
+  backend: block
+  createoptions: -q
+  csi.storage.k8s.io/fstype: xfs
+provisioner: pure-csi
+reclaimPolicy: Delete
+volumeBindingMode: Immediate
+allowVolumeExpansion: true
```

Comment:
Please add a blank line at the end of file

Reply:
Done